configuration/exodus-pipeline.yaml

```diff
@@ -46,6 +50,11 @@ Conditions:
     !Equals [!Ref env, stage]
   IsProd:
     !Equals [!Ref env, prod]
+  IsExodus:
+    !Equals [!Ref project, exodus]
+  IsExodusProd: !And
+    - !Condition IsExodus
+    - !Condition IsProd
```

Comment:
super minor: I'd prefer all the conditions have a consistent format

Reply:
By consistent, were you referring to not referencing another Condition, or consolidating the condition definition to a single line? 
```
  IsExodusProd: 
    !And [!Equals [!Ref project, exodus], !Equals [!Ref env, prod]]
```
vs 
```
  IsExodusProd: 
    !And [!Condition IsExodus, !Condition IsProd]
```
If the former, is there a reason why we shouldn't reference another condition?

Reply:
I was referring to the latter, single vs multi-line, just because all other conditions in the repo are formatted that way. Your second example looks great.